<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DasdPropertiesClass</class>
 <widget class="QWidget" name="DasdPropertiesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DasdProperties</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>DASD</string>
   </property>
   <widget class="QComboBox" name="typeCombo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>2305</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2311</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2314</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3330</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3340</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3350</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3375</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3380</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3390</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9345</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3310</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3370</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9332</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9335</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9336</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0671</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceNumber">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Type</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>431</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QRadioButton" name="localRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Local file</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="inetRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>INET Address</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="filename">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="filenameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseShadowButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shadow">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="shadowLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shadow File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nosynchioCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Non-synchroneus I/O</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ioModeGroupbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>I/O Mode</string>
    </property>
    <widget class="QRadioButton" name="standardRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Standard R/W I/O</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="ReadonlyRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Read-only I/O</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fakeWritesRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Fake writes</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="cuComboBox">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>290</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>(default)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2841</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2314</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3880</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3990</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3990-3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3990-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9343</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="cuLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Override Default Control Unit:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remotehost">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>HHH.HHH.HHH.HHH; </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="portSpinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>65536</number>
    </property>
    <property name="value">
     <number>3990</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="devnoSpinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QLabel" name="remotehostLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote Host</string>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="devnoLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Relative Device Number</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
